python/pyspark/sql/streaming/proto/StateMessage_pb2.py

```diff
@@ -17,8 +17,8 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # NO CHECKED-IN PROTOBUF GENCODE
-# source: StateMessage.proto
-# Protobuf Python Version: 5.27.3
+# source: org/apache/spark/sql/execution/streaming/StateMessage.proto
+# Protobuf Python Version: 5.28.3
```

Comment:
@jingz-db @bogao007 could you please take a look at the changes in generated files? since you touched these files recently

Reply:
The changes LGTM, thanks!